website/static/js/koHelpers.js

```diff
@@ -259,6 +260,15 @@ ko.bindingHandlers.tooltip = {
     init: tooltip,
     update: tooltip
 };
+
+var clipboard = function(el, valueAccessor) {
+    makeClient($(el));
+    $(el).attr("data-clipboard-text", valueAccessor());
```

Comment:
To be safe, it'd be good to handle both observables and non-observable values passed to this. `ko.unwrap` should help you out with that.